CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [#1992](https://github.com/FuelLabs/fuel-core/pull/1992): Parse multiple relayer contracts, `RELAYER-V2-LISTENING-CONTRACTS` env variable using a `,` delimiter.
 
 #### Breaking
+- [#2001](https://github.com/FuelLabs/fuel-core/pull/2001): Prevent GraphQL query body to be huge and cause OOM. The default body size is `256KB`. The limit can be changed by the `graphql-max-queries-body-limit` CLI argument.
```

Comment:
Currently we support queries up to 7.5mb at the load balancer level. Should we match that with our default here?

I think it's pretty common for RPC providers to support up to 10+ mb responses on other jsonrpc based chains.

cc @mchristopher

Reply:
This limit applies only to the query itself; it doesn't affect the response.

But I realized that we use a hex representation of transactions, so we need to support twice 256 KB. Maybe it is better to set it to 1 MB.